composer.json

```diff
@@ -19,9 +19,13 @@
         "composer/installers": "^1.2",
         "drupal-composer/drupal-scaffold": "^2.2",
         "cweagans/composer-patches": "~1.0",
-        "drupal/core": "~8.0",
-        "drush/drush": "~8.0",
-        "drupal/console": "~1.0"
+        "drupal/core": "~8.2",
+        "drush/drush": "8.1.3",
```

Comment:
Yeah, good question, we keep it locked at `8.1.3` due to some issues with Symfony errors in versions past this point.